### PR TITLE
Override firebase authentication service for local environment

### LINKED ISSUE #13: Override firebase authentication service locally

### PR DESCRIPTION
**Pull request by abarghoud**
Resolves #13 

This allows to bypass firebase authentification locally, it lets any user with any password login and stores everything in memory.

### CODE REVIEW

File: apps/client/src/app/app.tsx [hunk 1]
```diff
@@ -32,33 +35,38 @@ export function App() {
   const clientRecaptchaChecker: IRecaptchaChecker = isLocalEnvironment ?
     new LocalRecaptchaChecker() :
     new ClientRecaptchaChecker(axiosInstance);
-  const firebaseAuthenticationService = isLocalEnvironment ?
+  const authenticationAuthenticationService = isLocalEnvironment ?
```
Comment (Romain-Portanguen) [inline apps/client/src/app/app.tsx]: The naming is wrong. `authenticationAuthenticationService` ? 😂

File: apps/client/src/app/app.tsx [hunk 2]
```diff
@@ -32,33 +35,38 @@ export function App() {
...
     new LocalAuthenticationService() :
     new FirebaseAuthenticationService(clientRecaptchaChecker, googleRecaptchaGenerator);
+  const contactFormApiService = isLocalEnvironment ?
+    new LocalContactFormApiService() :
+    new FirebaseContactFormApiService();
   const ppsCertificateApiService = new PPSCertificateApiService(axiosInstance, googleRecaptchaGenerator)
 
   return (
-    <AuthenticationContext.Provider value={firebaseAuthenticationService}>
+    <AuthenticationContext.Provider value={authenticationAuthenticationService}>
```
Comment (Romain-Portanguen) [inline apps/client/src/app/app.tsx]: The same applies here. The global correction will fix this occurrence.

**Review by Romain-Portanguen — APPROVED** [review]
One small correction otherwise perfect 🤩

File: apps/client/src/app/app.tsx (hunk 1 above)
Comment (abarghoud) [reply apps/client/src/app/app.tsx]: Hahhaha, it was firebaseAuthenticationService, and I wanted to make it abstract from firebase, I renamed it without looking 😂, that's what we call abstraction overwhelming

**Pull request merged**